src/core/credentials/transport/tls/grpc_tls_certificate_provider.h

```diff
@@ -202,11 +194,8 @@ class FileWatcherCertificateProvider final
   PemKeyCertPairList pem_key_cert_pairs_ ABSL_GUARDED_BY(mu_);
   // The most-recent root data. It will be empty if the most recent read
```

Comment:
Please document the possible values:
- null if unset
- non-OK status if invalid
- non-null if valid